Islandora/islandora_defaults
# Adding new MODS fields as per Metadata Interest Group mappings

**GitHub Issue**: (link)
* https://github.com/Islandora-CLAW/CLAW/issues/1083

# What does this Pull Request do?
This PR adds various fields needed for a default MODS form as per the above noted ticket.  

# How should this be tested?
* SSH into the vagrant
* Go to Features: http://localhost:8000/admin/config/development/features
* Select Islandora bundle, click newly detected, import changes
* Add a vocabulary list item if needed in the taxonomy
* Create a new repository item
* Verify that new fields appear in the form and display

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora-CLAW/committers
